utils/wasm-gen/src/lib.rs

```diff
@@ -649,6 +653,26 @@ impl<'a> WasmGen<'a> {
         )
     }
 
+    pub fn gen_handle_reply(&mut self, module: Module) -> (Module, bool) {
+        if self.config.skip_handle_reply.get(self.u) {
+            return (module, false);
+        }
+
+        let funcs_len = module
+            .function_section()
+            .map_or(0, |funcs| funcs.entries().len() as u32);
+
+        if funcs_len == 0 {
+            return (module, false);
+        }
+
+        let func_no = self.u.int_in_range(0..=funcs_len - 1).unwrap();
```

Comment:
Isn't `0..=funcs_len - 1` equal to `0..funcs_len`?
```suggestion
        let func_no = self.u.int_in_range(0..funcs_len).unwrap();
```

Reply:
You're right, thanks.

Reply:
ops it requires inclusive range )